### DISCUSSION TITLE
Release 1.23.0

### DISCUSSION THREAD

**Discussion by jpkrohling** [jaegertracing/jaeger#3053] [General]
@jaegertracing/jaeger-maintainers, given that the Jaeger Agent memory consumption problem caused by Thrift should be mitigated, I think we should release a new minor version. Is there anything you'd like to have before releasing v1.23.0?

**Comment by yurishkuro** [comment]
sgtm

**Comment by jpkrohling** [comment]
I'm reviewing and merging some UI PRs bumping versions, some of them as a result of security fixes. I don't think we are affected by any of them (so far), but I'll try to clean up the PR queue for the UI a bit.

**Reply by jpkrohling** [reply]
PR to release the UI: https://github.com/jaegertracing/jaeger-ui/pull/764

**Reply by yurishkuro** [reply]
Don't forget to release the docs

**Reply by jpkrohling** [reply]
I will probably do it tomorrow, or Monday at the latest.
